server/vulnerabilities/nvd/cve_test.go

```diff
@@ -381,6 +381,14 @@ func TestTranslateCPEToCVE(t *testing.T) {
 		"cpe:2.3:a:simple_password_store_project:simple_password_store:1.7.0:*:*:*:*:macos:*:*": {
 			includedCVEs: []cve{{ID: "CVE-2018-12356", resolvedInVersion: "1.7.2"}},
 		},
+		"cpe:2.3:a:python:python:3.14.0a2:*:*:*:*:windows:*:*": {
+			includedCVEs: []cve{
+				{
+					ID:                "CVE-2024-12254",
+					resolvedInVersion: "3.12.9",
```

Comment:
This feels like it should be `3.14.0a3`, per https://www.cve.org/CVERecord?id=CVE-2024-12254

Reply:
@iansltx could this be an issue with the semver changes we made recently?

Reply:
Not sure. Would probably require digging into the vuln feed to see the information that's there. As mentioned in chat, I think that the CPE wouldn't even generate without the recent fix in place, but maybe we're matching the wrong CPE?

Reply:
We are no longer using semver for the resolved in version stuff >
https://github.com/fleetdm/fleet/blob/main/server/vulnerabilities/nvd/cve.go#L678-L702
Using the custom nvdtools code ^